ANSWERS.md

```diff
@@ -0,0 +1,30 @@
+- What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+
+****Props are the mechanism React uses to let components communicate with each other.
+
+
+- Describe a life-cycle event in React?
+
+
+**The Birth/Mounting Phase
+
+**Growth/Updating Phase
+
+**Death/Un-mounting Phase
+
+
+
+- Explain the details of a Higher Order Component?
+
+** It shares the same functionality across multiple components.
+
+
+- What are three different ways to style components in React? Explain some of the benefits of each.
+
+*CSS Stylesheet
+
+*Inline styling
+
+*CSS Modules
+
+*Styled-components
```

Comment:
Thank you for these! Make sure you understand the benefits of each way to style components!